helm/nessie/templates/ingress.yaml

```diff
@@ -39,7 +39,7 @@ spec:
         paths:
           {{- range .paths }}
           - path: {{ . }}
-            pathType: ImplementationSpecific
+            pathType: {{ default "ImplementationSpecific" $.Values.ingress.pathType }}
```

Comment:
Could you add the new option `ingress.pathType` to values.yaml and document it please? Otherwise it looks like a "hidden feature" 🙂

Reply:
You're right, done this.
Actually, I'd prefer to Prefix by default, but for backward compatibility, I left it as it was.